packages/api/index.mts

```diff
@@ -1,5 +0,0 @@
-import app from './server/http.mjs';
```

Comment:
I think this will break the build

Reply:
Yes it will... whoops, didn't meant to do this!